doc/source/release-notes/Percona-XtraDB-Cluster-5.6.47-28.40.rst

```diff
@@ -0,0 +1,24 @@
+.. _PXC-5.6.47-28.40:
+
+================================================================================
+*Percona XtraDB Cluster* 5.6.47-28.40
+================================================================================
+
+:Date: May 4, 2020
+:Installation: `Installing Percona XtraDB Cluster <https://www.percona.com/doc/percona-xtradb-cluster/5.6/installation.html>`_
+
+Improvements
+================================================================================
+
+* :jirabug:`PXC-3006`: Added information about xinetd 'cps' option in PXC HAProxy documentation
+* :jirabug:`PXC-2602`: Added Ability to Configure xbstream options with wsrep_sst_xtrabackup-v2
```

Comment:
Jira ticket only mentions 5.7, please also update that too.

Reply:
Added 5.6 to the ticket.